src/Uno.UI/UI/Xaml/Media/VisualTreeHelper.cs

```diff
@@ -193,11 +193,34 @@ public static FrameworkElement AdaptNative(_View nativeView)
 					$"Use {nameof(TryAdaptNative)} if it's not known whether view will be native.");
 			}
 
-			return new ContentPresenter
+			var host = new ContentPresenter
 			{
 				IsNativeHost = true,
 				Content = nativeView
 			};
+
+			// Propagate layout-related attached properties to the managed wrapper, so the host panel takes them into account
+			PropagateAttachedProperties(
```

Comment:
This won't make the DP's follow value changes, but it's probably better than the alternative for now.

Reply:
Yeah tbh I wasn't sure how to bind to an attached property value as the target path? This approach covers the most common use case.